packages/ETLoad.package/ETLSampleImageSegmentTest.class/instance/testFile..st

```diff
@@ -1,3 +1,9 @@
 testing
 testFile: aFile
-	Transcript show: aFile; cr.
\ No newline at end of file
+	| reference |
+	
+	reference := ETLProjectLoadHelper loadSampleImageSegmentReference: aFile.
+	
+	[ ETLSegmentLoader do: [ ETLProjectLoadHelper loadSampleImageSegment: aFile. ]
+			withLoadResultDo: [ :result | self assert: (result literalEqual: reference). ]. ] ensure: [
```

Comment:
I'll add a comment later, but the difference between `=:` and `literalEqual:` is that `literalEqual` also ensures that two values are of the same class.